src/locales/en_US.js

```diff
@@ -193,6 +193,8 @@ const strings = {
   settings_account_title_cap: 'Account',
   settings_button_change_password: 'Change Password',
   settings_button_debug: 'Debug',
+  settings_turn_on_developer_mode: 'Turn On Developer',
```

Comment:
Change text to 'Enable Developer Mode' and 'Disable Developer Mode'.

Shrink text if needed to ensure this fits on an iPhone 5 width screen